samples/automl/automlVisionDataset.js

```diff
@@ -0,0 +1,400 @@
+        console.log(`Dataset example count: `, dataset[i].exampleCount);
+        console.log(`Image Classification type:`);
+        console.log(
+          `\t`,
+          dataset[i].imageClassificationDatasetMetadata.classificationType
+        );
+        console.log(`Dataset create time: `);
+        console.log(`\tseconds: `, dataset[i].createTime.seconds);
+        console.log(`\tnanos: `, dataset[i].createTime.nanos);
+        console.log(`\n`);
+      }
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_listDatasets]
+}
+
+function getDataset(projectId, computeRegion, datasetId) {
+  // [START automl_vision_getDataset]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `numeric Id of the project from PROJECT_ID env variable, e.g. 123456`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset, e.g. "ICN3874392874098"`;
+
+  // Get the full path of the dataset.
+  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+
+  // Get complete detail of the dataset.
+  client
+    .getDataset({name: datasetFullId})
+    .then(responses => {
+      const dataset = responses[0];
+
+      // Display the dataset information.
+      console.log(`Dataset name: `, dataset.name);
+      console.log(`Dataset Id: `, dataset.name.split(`/`).pop(-1));
+      console.log(`Dataset display name: `, dataset.displayName);
+      console.log(`Dataset example count: `, dataset.exampleCount);
+      console.log(
+        `Classification type: `,
+        dataset.imageClassificationDatasetMetadata.classificationType
+      );
+      console.log(`Dataset create time: `);
+      console.log(`\tseconds: `, dataset.createTime.seconds);
+      console.log(`\tnanos: `, dataset.createTime.nanos);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_getDataset]
+}
+
+function importData(projectId, computeRegion, datasetId, path) {
+  // [START automl_vision_importData]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `numeric Id of the project from PROJECT_ID env variable, e.g. 123456`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const datasetId = `Id of the dataset, e.g. "ICN3874392874098"`;
+  // const path = `Cloud Storage Uri, target must be in AutoML Vision CSV format, e.g. “gs://myproject/mytraindata.csv”;`
+
+  // Get the full path of the dataset.
+  const datasetFullId = client.datasetPath(projectId, computeRegion, datasetId);
+
+  // Get the multiple Google Cloud Storage URIs.

```

Comment:
As far as I can tell, this sample does not accept multiple URIs

```js
function importData(projectId, computeRegion, datasetId, path) { // <---- simple 'path' not 'paths'
  // ....
  // const path = `Cloud Storage Uri, target must be in AutoML Vision CSV format, e.g. “gs://myproject/mytraindata.csv”;`
  // ^---- this makes no mention of multiple paths
```

If the sample must accept a list of GCS URIs, update the parameter name and update the description in the TODO placeholder

The current TODO description shows it as a single URI